src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -57,18 +62,14 @@
     Message getDisplayName();
 
     /**
-     * Returns whether the {@link Player} can fly via the fly key.
+     * Sets the player's display name.
      *
-     * @return {@code True} if the {@link Player} is allowed to fly
-     */
-    boolean getAllowFlight();
-
-    /**
-     * Sets if the {@link Player} can fly via the fly key.
+     * <p>Passing {@link com.google.common.base.Optional#absent()} will set the
```

Comment:
You need to fix this for `Nullable`, and also add a note about this to the parameter description (like `The new display name, or null to reset it`